common/src/test/java/io/netty/util/concurrent/FastThreadLocalTest.java

```diff
@@ -76,6 +80,30 @@ public void run() {
         }
     }
 
+    @Test
+    public void testMultipleSetRemove() throws Exception {
+        final FastThreadLocal<String> threadLocal = new FastThreadLocal<String>();
+
+        final Field liveSetField = ObjectCleaner.class.getDeclaredField("LIVE_SET");
+        liveSetField.setAccessible(true);
+        final Set<?> liveSet = (Set<?>) liveSetField.get(ObjectCleaner.class);
+        final int sizeWhenStart = liveSet.size();
```

Comment:
can we make this work without reflection ? Maybe expose this as a getter on the `ObjectCleaner` ?